packages/suite/src/components/onboarding/ConnectDevicePromptManager/components/UnexpectedDeviceState.tsx

```diff
@@ -39,12 +40,14 @@ export const UnexpectedDeviceState = ({
                     {
                         key: 'bootloader',
                         heading: <Translation id="TR_RECONNECT_IN_NORMAL" />,
-                        description:
-                            trezorModel === 1 ? (
-                                <Translation id="FIRMWARE_CONNECT_IN_NORMAL_MODEL_1" />
-                            ) : (
-                                <Translation id="FIRMWARE_CONNECT_IN_NORMAL_MODEL_2" />
-                            ),
+                        description: (
+                            <Translation
+                                id={selectByDeviceModel(deviceModel, {
```

Comment:
A flag or a config with touch / buttons could be used here as well. Seems like a useful categorisation also because it's a part of the new internal naming scheme